source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -81,6 +81,11 @@ def getSummaryStatsFromCAdvisor(winNode)
       return getResponse(winNode, relativeUri)
     end
 
+    def getCongifzCAdvisor(winNode)
+      relativeUri = "/configz"
```

Comment:
do we have any restriction support of this endpoint on  k8s version? or will this api work on k8s version 1.17 or up?

Reply:
For secure cadvisor, will this require adding configz resource for omsagent-reasder resource?

Reply:
tested it in a cluster where isSecure is set to true and it works. resolving conversation. Please reopen it if needed.